save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/Plugin.kt

```diff
@@ -91,7 +92,11 @@ abstract class Plugin(
             val foundTests = rawTestFiles.filter { rawTestFile ->
                 testFiles.any {
                     if (fs.exists(it.toPath())) {
-                        it in rawTestFile.test.toString()
+                        (it.toPath().parents().toList() + listOf(it.toPath())).all { testFile ->
+                            (rawTestFile.test.parents().toList() + listOf(rawTestFile.test)).any { rawTestFileDir ->
+                                testFile.toString() == rawTestFileDir.toString()
```

Comment:
Is this the logic we are aiming for, when they have at least one common parent directory?

Reply:
Not really. All parent directories of testFiles submitted by the user must match at least one parent directorie rawTestFiles

Reply:
We can compare parents of `rawTestFile` with entire `testFile`

Reply:
Fixed